android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -1208,7 +1208,12 @@
 	 * @module.api
 	 */
 	public static final String PROPERTY_FONT_SIZE = "font-size";
-
+	
+	/**
+	 * @module.api
+	 */
+	public static final String PROPERTY_MINIMUMFONTSIZE = "minimumFontSize";
```

Comment:
Might want to change `PROPERTY_MINIMUMFONTSIZE ` to `PROPERTY_MINIMUM_FONT_SIZE` to more align with Appcelerator's coding style.